sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -170,5 +180,12 @@ class VersionsSuite extends SparkFunSuite with Logging {
         false,
         false)
     }
+
+    test(s"$version: create index and reset") {
+      client.runSqlHive("CREATE TABLE indexed_table (key INT)")
+      client.runSqlHive("CREATE INDEX index_1 ON TABLE indexed_table(key) " +
+        "as 'COMPACT' WITH DEFERRED REBUILD")
+      client.reset()
+    }
```

Comment:
Some of the other tests do verify the results (i.e. create a table and then make sure its in the list of tables) and in general I'm skeptical of the argument that just because existing tests are great, new ones also shouldn't be.  That said, we can merge this and I can improve the tests in a followup.